OptimizelySDK/Properties/AssemblyInfo.cs

```diff
@@ -40,9 +40,6 @@
 //      Build Number
 //      Revision
 //
-// You can specify all the values or you can default the Build and Revision Numbers 
-// by using the '*' as shown below:
-// [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("3.11.2.1")]
-[assembly: AssemblyFileVersion("3.11.2.1")]
-[assembly: AssemblyInformationalVersion("3.11.1")] // Used by Nuget.
```

Comment:
I canNOT believe I missed the version bump to 3.11.3 here in the main SDK.  

This needs automation (at least for checking) I think.